documentation/src/main/resources/_posts/2023-08-23-ditto-benchmark.md

```diff
@@ -0,0 +1,465 @@
+---
+title: "Eclipse Ditto Benchmark"
+published: false
+permalink: 2023-08-23-ditto-benchmark.html
+layout: post
+author: vvasilevbosch
+tags: [blog]
+hide_sidebar: true
+sidebar: false
+toc: true
+
+---
+
+This blog post is presenting a benchmark of Eclipse Ditto. It consists of a few scenarios to cover most important ditto functionalities, test the performance and provide a tuning guide. This benchmark is done via the [benchmark-tool](https://github.com/bosch-io/ditto/tree/master/benchmark-tool), based on [k6](https://k6.io/) load testing tool
```

Comment:
The link to the "benchmark-tool" points to the Bosch-IO fork - should be changed to Ditto main repo.